src/App.js

```diff
@@ -22,9 +24,17 @@ function App() {
           <NavBar />
           <Content style={{ background: "white", padding: "0 5px" }}>
             <div className="site-layout-content">
+              <Route
+                exact
+                path="/story/:id"
+                render={(props) => <BlogCard {...props} />}
+              />
+              <Route path="/coupons" component="#" />
+              <Route path="/blog" component={BlogCards} />
+              <Route path="/admin" component="#" />
+              <Route path="/cart" component="#" />
```

Comment:
Are these route duplicates? Because I see admin cart and couponspage route imported correctly at 37,38 and 39th lines.